ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -1131,6 +1131,22 @@ tests:
         ISO_IMAGE_TYPE=minimal-iso
       PLATFORM: nutanix
     workflow: assisted-nutanix
+- as: e2e-ai-operator-ztp-remove-node
```

Comment:
```suggestion
- as: e2e-ai-operator-ztp-remove-node-periodic
```

Reply:
also can you add yourself to bugmaster configuration to get ping when the job fails? https://github.com/openshift-assisted/bugmaster-configurations/blob/main/config/channel/assisted-deployment-ci/bug_master_configuration.yaml#L7

Reply:
I had this originally but the job is already prefixed with `periodic-` so it seemed redundant.

Reply:
we have a mix of presubmits and periodics in this file, we cannot have the same job names even though they are different types of jobs. That's why we suffix all the periodics.

Reply:
Ah interesting. So you can't have a single job definition be used for both a periodic and persubmit? That seems odd.